tests/mpd/protocol/__init__.py

```diff
@@ -80,3 +80,19 @@ def assertNotInResponse(self, value):  # noqa: N802
     def assertEqualResponse(self, value):  # noqa: N802
         self.assertEqual(1, len(self.connection.response))
         self.assertEqual(value, self.connection.response[0])
+
+
+class BaseTestCaseMixerNone(BaseTestCase):
```

Comment:
Don't bother with using the `BaseTest` pattern here as we only have the one test case using this setup logic.

What I was thinking of was more something like the following snippet, mainly because it saves you having to re-implement the rest of the setup method.

```
class BaseTestCase(unittest.TestCase):
    enable_mixer = True

    def setUp(self):  # noqa: N802
        if self.enable_mixer:
            self.mixer = dummy_mixer.create_proxy()
        else:
            self.mixer = None
```